project.md

```diff
@@ -58,11 +81,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I beleive that a person working on this project would spend the most time with system aspects but It would depend on how much natural language is used for the laguage itself. If the laguage is 100% natural laguage driven there will have to be many componets used for laguange analysis. The sysytem processes themselves will be roughly straight forward. build a data structire to score stats per game per player, use those stats to make calculations on averages, team production, and other analysis. Then present all findings per game to the user once an end game command is enetered. Additionally allow for data to be uploaded or saved for people like familly memebers or program websites to see. This process would grow more difficult as you add more sports but could be failry simple for one or two sports. 
+
```

Comment:
I think that there are going to be many more design decisions to be made here than anticipated. An example of something you might have to consider in the future is statement order, so what if someone wants to say "20 yard pass to player number 82 from player number 11" or in shorthand form "20y P to 82 from 11"? I think testing out more ideas like this might require a significant amount of thinking which even if not all of them translate directly to code, make this project require more design decisions than the current estimate of primarily system implementations above.